### ISSUE TITLE
[Request]: 

### ISSUE THREAD

**Issue by leo-arch** [ScoopInstaller/Main#4626]
### Prerequisites

- [X] I have searched all issues/PRs to ensure it has not already been reported or fixed.

### Criteria

- [X] Non-GUI tool
- [X] Reasonably well-known and widely used (e.g. if it's a GitHub project, it should have at least 500 stars and/or 150 forks)
- [X] English interface (or at least English documentation)
- [X] Latest stable version
- [X] Full version (i.e. not a trial version)
- [X] Fairly standard install (e.g. uses a version-specific download URL, no elaborate pre/post install scripts)

### Name

clifm

### Description

The shell-like, command line file manager

### Homepage

https://github.com/leo-arch/clifm

### Download Link(s)

https://github.com/leo-arch/clifm/releases/tag/v1.11

### Some Indication of Popularity/Repute

**Clifm** has almost 1K starts on Github, and is already packaged for several Linux/Unix systems, like Alpine, Fedora, NixOS, Termux, Gentoo, FreeBSD, DragonFly, and MacOS, including Cygwin. It has been reviewed by [Brodie Robertson](https://www.youtube.com/watch?v=CJmcisw9F90) and is ranked among the 2023 top 5 console file managers for Linux by [linuxlinks.com](https://www.linuxlinks.com/BestConsoleFileManagers).

**Comment by rashil2000** [comment]
Are libraries required during runtime?

The best way would be to have a single statically linked binary, along with Readme, License etc. files in an archive.

**Comment by leo-arch** [comment]
Yes, there are libs required at runtime (by default we dynamically link libs), at the very least `readline`.

Is there a way to check required files locally? Maybe a guideline to build packages for Scoop? I normally use Cygwin to test **clifm** on Windows systems.

**Comment by rashil2000** [comment]
On a Windows machine you can just run:
```console
scoop install <local-path-to-json>
```
to install the package, which you can then test locally.

**Comment by leo-arch** [comment]
Thanks @rashil2000! I'll check this out and let you know as soon as I have a working manifest.

**Comment by amreus** [comment]
@leo-arch the source code is very unix/linux focused.  I was able to compile `clifm` using cygwin64, but there are some things that do not work properly.  See the following .zip for executable and a slew of required cygwin .dll files.

[clifm.zip](https://github.com/ScoopInstaller/Main/files/14001587/clifm.zip)

I was not able to compile `clifm` using `msys2`, but I am not sure if the problem is with `msys`, the source code, or my setup.

If you want to support Windows, you might see if there is some cross-platform library to handle the features you need.

**Comment by leo-arch** [comment]
Hi @amreus. Please take a look at both the [install instructions](https://github.com/leo-arch/clifm/wiki/Introduction#compilinginstalling-from-source) and [this issue](https://github.com/leo-arch/clifm/issues/162).

I've been testing **clifm** on Cygwin and MinGW since 2 years ago aprox (including some users feedback), and it works reasonably well (it's true though that `the source code is very unix/linux focused`).

Could you please specify what is not working properly?

**Comment by amreus** [comment]
@leo-arch Ok followed those instructions - I didn't notice the Makefiles at first.

Outside of msys2, the executable still requires a bunch of .sll files.  I don't think there is a way around it.

I want to be able to run apps in cmd.exe or powershell.  clifm doesn't find its support files (help file, shell scripts) when executed in this environment (not in msys2.)  Maybe just a matter of making clifm look in a relative location for its files since Windows doesn't used /usr/local/*. 

Anyway, thanks for the app - pretty cool.

**Comment by leo-arch** [comment]
> Outside of msys2, the executable still requires a bunch of .sll files. I don't think there is a way around it.

**Clifm** won't run _natively_ on Windows (and it probably never will), but only on Cygwin-like Unix/Linux environments.

> Maybe just a matter of making clifm look in a relative location

**Clifm** looks for executables in those paths specified in the **PATH** environment variable. You may try adding whatever path you need here.
